eo-phi-normalizer/src/Language/EO/Phi/Dataize.hs

```diff
@@ -270,87 +278,129 @@ evaluateBytesBytesFunChain name = evaluateUnaryDataizationFunChain name intToByt
 evaluateFloatFloatFloatFunChain :: String -> (Double -> Double -> Double) -> Object -> EvaluationState -> DataizeChain (Object, EvaluationState)
 evaluateFloatFloatFloatFunChain name = evaluateBinaryDataizationFunChain name floatToBytes bytesToFloat wrapBytesInFloat extractRho (extractLabel "x")
 
+knownAtoms :: [(String, String -> Object -> EvaluationState -> DataizeChain (Object, EvaluationState))]
+knownAtoms =
+  [ ("Lorg_eolang_int_gt", \name obj -> evaluateIntIntBoolFunChain name (>) obj)
```

Comment:
If you change the order of arguments, you can avoid lambdas and simply write `evaluateIntIntBoolFunChain (>)`.

Reply:
Addressed in https://github.com/objectionary/normalizer/pull/426/commits/32be6c98f22e44833723d70e3c27f0607e56cdd5